code/mob.dm

```diff
@@ -219,6 +219,11 @@
 
 	var/last_pulled_time = 0
 
+	/// stores total accumulated radiation dose
+	var/radiation_dose = 0
+	/// natural decay of radiation exposure
+	var/radiation_dose_decay = 0.001
```

Comment:
the decay rate seems comically small for the average round length. If you take a 0.4sv dose (the upper end of the lowest damage scale for rads), you will take damage 200 times before the rads decay to safe levels (taking ~140 tox damage over those 12 minutes or so minutes) barring medical intervention.